TI-reports/2025/2025-Q1-GCP-WG.md

```diff
@@ -0,0 +1,56 @@
+
+We are also exploring folding in the "CRA tech bi-weekly" call into the remit of this group.
+
+Since forming, we have spent a fair amount of time refining the charter, in particular the wording around how we open up the group as much as possible while also making it clear who gets to vote when and if a vote is required.
+
+We now have a schedule of calls for our SIGs and have started to take minutes in our main minutes doc. We are operating in a similar mode to the Best Practices Working group, with our SIGs reporting into the main working group call. 
+
+We held a special session at LF Member Summit which is also minuted in our minutes doc.
+
+## Awareness SIG
+
+The awareness SIG is the furthest along. It's led by [Megan Knight](https://github.com/businesscasualkesha), Arm. The scope is activities that drive awareness of the work of this group and of the regulatory landscape in general. The SIG has been marshalling blog posts, upcoming conference schedule, as well as the CRA introductory course.
+
+Blog Posts:
+* [Linux Foundation Europe and OpenSSF Launch Initiative to Prepare Maintainers, Manufacturers, and Open Source Stewards for Global Cybersecurity Legislation
+](https://openssf.org/press-release/2025/01/31/linux-foundation-europe-and-openssf-launch-initiative-to-prepare-maintainers-manufacturers-and-open-source-stewards-for-global-cybersecurity-legislation/)
+* [What Will My Business Need to do for the CRA?](https://openssf.org/blog/2025/03/24/what-will-my-business-need-to-do-for-the-eu-cra/)
+* [Does the EU CRA affect my business?](https://openssf.org/blog/2025/02/20/does-the-eu-cra-affect-my-business/)
+* [OpenSSF Policy Summit DC 2025 Recap](https://openssf.org/blog/2025/03/14/openssf-policy-summit-dc-2025-recap/)
+
+## Tooling SIG
+
+The Tooling SIG has is still in start-up mode. However we have one lead, [Puerco](https://github.com/puerco), and we're looking for a co-lead. The scope is to coordinate work on relevant tooling and processes. What tools already exist out there that can help maintainers, stewards and manufacturers? What additional features do we need from existing tools?
+
+## Standards SIG
+
+The Standards work stream is progressing. Currently we have one lead [[Tobias](https://github.com/0xAverageUser)](https://github.com/0xAverageUser), we are actively seeking a co-lead.
+The SIGs mission was defined as: 
+
+> The OpenSSF Global Cyber Policy WG - Standardisation SIG’s mission is to align regulatory (e.g., the EU CRA) compliance strategies & standards across open-source participants to ensure clarity, consistency, and industry-wide adoption and coordination. Participants include stewards, projects, and maintainers.
+
+To meet that objective, we will likely start developing an index of curated Minutes, Resources, and References from OpenSSF, Eclipse, and OpenJS, as a "Google for CRA."
```

Comment:
probably just a nit-pick / bike-shedding, but for my understanding: "Google for CRA" refers to something like a CRA-related landscape?

Reply:
No, it's about building, maintaining and deploying a database, (an index) for CRA resources which can be queried.

Reply:
I think it'd be good to spell that out because I must admit I didn't understand what "a Google for CRA" meant either.

Reply:
Updated

Reply:
did you forget to push your commit? It still reads: as a "Google for CRA."

Reply:
suggested the change here: https://github.com/ossf/tac/pull/462/files#r2022763887

Reply:
Ok, I see it now, thanks. It's not been applied though. @torgo?